index.html

```diff
@@ -28,14 +28,6 @@
             </ul>
```

Comment:
Please remove <ul> tag. this tag is not required for now: since this part of the code is not used, it makes your codebase unclean.

Reply:
- [ ] As I mentioned before your comment should be a suggestion instead of making it look like an order.  For example, you can say, "**_It would be better_** to remove this tag. This tag is not required since this part of the code is not used, it makes your codebase unclean."

Reply:
It would be better to remove the <ul> tag; since this part of the code is currently not used,